diff --git a/CMakeLists.txt b/CMakeLists.txt
index 23bd9bba4..c4577b8df 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,7 +38,6 @@ enable_language(CXX)
 
 # TODO(crbug,com/389897612): Register tests with CTest. For now, we include this
 # only to pick up the standard |BUILD_TESTING| option.
-include(CTest)
 include(GNUInstallDirs)
 
 set(INSTALL_ENABLED 1)
@@ -649,15 +648,9 @@ add_library(ssl ${SSL_SOURCES})
 set_property(TARGET ssl PROPERTY EXPORT_NAME SSL)
 target_link_libraries(ssl crypto)
 
-add_library(decrepit ${DECREPIT_SOURCES})
-target_link_libraries(decrepit ssl crypto)
-
 if(APPLE)
   set(PKI_CXX_FLAGS "-fno-aligned-new")
 endif()
-add_library(pki ${PKI_SOURCES})
-target_link_libraries(pki crypto)
-target_compile_options(pki PRIVATE ${PKI_CXX_FLAGS})
 
 if(BUILD_TESTING)
   add_library(test_support_lib STATIC
@@ -710,9 +703,6 @@ if(FIPS)
   target_link_libraries(entropy_modulewrapper crypto)
 endif()
 
-add_executable(bssl ${BSSL_SOURCES})
-target_link_libraries(bssl ssl crypto)
-
 if(FUZZ)
   if(LIBFUZZER_FROM_DEPS)
     file(GLOB LIBFUZZER_SOURCES "util/bot/libFuzzer/*.cpp")
@@ -798,7 +788,6 @@ endif()
 
 if(INSTALL_ENABLED)
   install(TARGETS crypto ssl EXPORT OpenSSLTargets)
-  install(TARGETS bssl)
   install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
   install(EXPORT OpenSSLTargets
           FILE OpenSSLTargets.cmake
